### PR TITLE
use env file for systemd service

### PR DESCRIPTION
**Pull request by trevarj**
Put command line arguments in environment file.

Closes https://github.com/hufrea/byedpi/issues/149

### CODE REVIEW

File: dist/linux/byedpi.service [hunk 1]
```diff
@@ -3,7 +3,8 @@ Description=byedpi
 Documentation=https://github.com/hufrea/byedpi
 
 [Service]
-ExecStart=ciadpi --split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s
+EnvironmentFile=/etc/default/byedpi
+ExecStart=ciadpi $OPTIONS
```
Comment (dartvader316) [inline dist/linux/byedpi.service:7]: Probably using something like `$BYEDPI_OPTIONS` will be better to make sure there is no problems with global environment variables.

File: dist/linux/README.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ systemctl --user enable byedpi.service
 systemctl --user start byedpi.service
 ```
 
+Optionally, create a config file with command line arguments:
```
Comment (dartvader316) [inline dist/linux/README.md:21]: Would be good to say where to create a config.

File: dist/linux/README.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ systemctl --user enable byedpi.service
...
+
+```sh
+OPTIONS=--split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s
```
Comment (dartvader316) [inline dist/linux/README.md:24]: Shell variable value should be put in quotation marks.

**Review by dartvader316 — APPROVED** [review]
LGTM

**Comment by lufog** [comment]
Alternative PR #152

**Comment by trevarj** [comment]
@lufog Oh sorry, I will close. Yours looks better! Thanks!

**Pull request closed without merging**